### DISCUSSION TITLE
AMD PCnet PCI cards and ISA+ unnecessary config

### DISCUSSION THREAD

**Discussion by jadasse** [86Box/86Box#2227] [Q&A] [answered]
Why do the network adapters listed in the title have the Configure button when no apparent config is needed?
![amdpcnet](https://user-images.githubusercontent.com/71288109/158254924-363eb262-ad07-415a-9618-7cee0e1522e6.png)

**Comment by dhrdlicka** [comment, marked as answer]
Network adapters have a hidden MAC address setting. The UI doesn't take that in account, and shows the Configure button whenever a device has a non-empty option array.
